ui/app/AppLayouts/Chat/stores/CommunityTokensStore.qml

```diff
@@ -66,6 +75,17 @@ QtObject {
         root.deployFeeUpdated(root.deployFee)
     }
 
+    function computeSelfDestructFee(chainId) {
+        // TODO BACKEND
+        root.selfDestructFeeUpdated("0,0005 ETH")
```

Comment:
Even if it's a fake model... this should not be a string with a hardcoded decimal separator to `,`